### ISSUE TITLE
Unable to pass extra flags

### ISSUE THREAD

**Issue by dnielso5** [yaturner/myWeatherPanel#2]
trying to run sudo ./weatherPanel.py -D --led-brightness=10 or any other variation to the code dose not work. am i missing anything?

**Comment by yaturner** [comment]
No, I don't actually pass the weatherpanel flags in to the led library. But
it would be easy to fix. I get the led flags from the config file

**Comment by yaturner** [comment]
Maybe I should add a new configuration parameter called ledflags,  what do
you think?

**Comment by dnielso5** [comment]
i tried adding them into the cfg under the [LED] header but they didn't go either.

adding a [ledflags] would probably be a good idea to be able to add extra configs. i have a very small flicker and being able to add flags would be a nice test to see why.

**Comment by yaturner** [comment]
I am on vacation, but i will look at it when I get home,  thank you for
your input

**Comment by dnielso5** [comment]
Ok, thank you.

**Comment by dnielso5** [comment]
So i was able to add in the extra arguments i needed by adding this to the .py script

```
            options.brightness = int(ledOptions.get("brightness", 100))
            options.gpio_slowdown = int(ledOptions.get("gpio_slowdown", 1))
            options.pwm_bits = int(ledOptions.get("pwm_bits", 11))

```

and then updating the .cfg file with the variables.

ALso you need to update the openweather link to 3.0 instead of 2.5 as they have updated.

The free account can not get API calls anymore, you have to sign up for the free 1000 daily calls + $0.15/100 calls after plan for it to work.

**Comment by yaturner** [comment]
Awesome, thanks for the update!
